app/models/piece.rb

```diff
@@ -86,32 +86,33 @@ def obstructed?(row_dest, col_dest)
         row_pos += 1
         (row_pos...row_dest).each do |row_num|
           col_pos += 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_pos).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_pos).nil?
         end
       # row and col both decrease
       elsif row_pos > row_dest && col_pos > col_dest
         row_dest += 1
         (row_dest...row_pos).each do |row_num|
           col_dest += 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_dest).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_dest).nil?
         end
       # row increases col decreases
       elsif row_pos < row_dest && col_pos > col_dest
         row_pos += 1
         (row_pos...row_dest).each do |row_num|
           col_pos -= 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_pos).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_pos).nil?
         end
       # row decreases col increases
       else
         row_dest += 1
         (row_dest...row_pos).each do |row_num|
           col_dest -= 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_dest).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_dest).nil?
         end
       end
     else
       return "ERROR! in is_obstructed? method line:83"
```

Comment:
@jonathanpike good pickup, this line was originally there just as an error handling event in case the code ended up at this line it would be easy to find out where things were going wrong (the line was originally line 83). I was going to ask Travis about error handling and what best practice is in these cases.